packages/lint/tsconfig.json

```diff
@@ -3,7 +3,9 @@
   "compilerOptions": {
     "composite": true,
     "outDir": "./lib",
-    "rootDir": "./src"
+    "rootDir": "./src",
```

Comment:
Please turn on `declaration` for the lint package, it is a library so its consumer may want to have typing for this.

Reply:
For declarations to work, `package.json` need to be updated.

You can refer to here to see how it works: https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package

Reply:
On that doc, it says that the `types` property is ignored if `files` is included. Does that mean that the current setup is fine?

Reply:
I'm not sure whether it will find the declaration file recursively in the `lib` directory specified by `files`. Can you test locally (e.g. using `npm link`) whether it works?

Reply:
I did a `npm publish` dry-run and all of the declaration files are included in the package

Reply:
I mean whether TypeScript would recognize the typing if you do `import { lintCommitMessage } from "@oursky/gitlint";`

Reply:
yep, the bot build works fine after lint package is linked